.github/workflows/test.yml

```diff
@@ -16,8 +16,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ${{ fromJSON(vars.TEST_PYTHON_VERSIONS) }}
-        os: ${{ fromJSON(vars.TEST_OS_VERSIONS) }}
+        python-version: ["3.11"]
+        os: ["ubuntu-latest"]
```

Comment:
Would something like this work?
`${{ fromJSON(vars.TEST_PYTHON_VERSIONS || '["3.11"]' ) }}`

Reply:
oh cool!! 🤩 ✨ 

no idea if it works, but I have been looking for something similar for quite some time 😅 

Though, I'm not sure if we want to go this route, or rather keep things more in `tox`, and for that less tight to GitHub infra, specially as we want to support GitLab as well.

Reply:
Although I like the idea of having a fallback, given that we want to move away from `vars.*` anyway it does not bring much...